src/pages/index.mdx

```diff
@@ -30,9 +30,9 @@ We've just scratched the surface. There is much more for you to discover! Go ahe
 <SectionBanner
   image
   to={"/extending-agent-app/"}
-  title={"Extending Agent App"}
+  title={"Extending Apps' interfaces"}
```

Comment:
As this section includes SDKs for both products, I decided to change the title here and adjust the description so it mentioned both products.

Reply:
I'd drop the `'` and make `apps` singular